### PR TITLE
Patch/yonatanari/disabled states correction

### PR DESCRIPTION
**Pull request by 3dyonic**
https://monday.monday.com/boards/245345663/pulses/2695136486

#### Basic
- [ ] Used plop (`npm run plop`) to create a new component.
- [ ] PR has description.
- [ ] New component is functional and uses Hooks. 
- [ ] Component defines [`PropTypes`](https://reactjs.org/docs/typechecking-with-proptypes.html).
#### Style
- [ ] Styles are added to `NewComponent.modules.scss` file inside of the `NewComponent` folder.
- [ ] Component uses CSS Modules.
- [ ] Design is compatible with [Monday Design System](https://design.monday.com/).
#### Storybook
- [ ] Stories were added to `/src/NewComponent/__stories__/NewComponent.stories.js` file.
- [ ] Stories include all flows of using the component.
#### Tests
- [ ] Tests are compliant with [TESTING_README.md](TESTING_README.md) instructions.

### CODE REVIEW

File: src/components/AttentionBox/__stories__/attentionBox.stories.mdx [hunk 1]
```diff
@@ -205,7 +205,7 @@ Provides cotextual and related information.
           <div className="monday-style-attention-box_search">
             <div className="monday-style-attention-box_inline-container">
               <Avatar size={Avatar.sizes.MEDIUM} src={person} type={Avatar.types.IMG} />
-              <span className="monday-style-attention-box_name">May Kishon<span>(UX/UI Product Designer)</span></span>
+              <span className="monday-style-attention-box_name">May Kishon <span>(UX/UI Product Designer)</span></span>
```
Comment (orrgottlieb) [inline src/components/AttentionBox/__stories__/attentionBox.stories.mdx:208]: why did this change?

Comment (orrgottlieb) [inline src/components/AttentionBox/__stories__/attentionBox.stories.mdx:208]: why did this change?

**Review by orrgottlieb — CHANGES_REQUESTED** [review]

File: src/components/AttentionBox/__stories__/attentionBox.stories.mdx (hunk 1 above)
Comment (3dyonic) [reply src/components/AttentionBox/__stories__/attentionBox.stories.mdx:208]: visual quirk Evgeniy Kazenic detected

Comment (3dyonic) [reply src/components/AttentionBox/__stories__/attentionBox.stories.mdx:208]: can change to &nbsp;

**Review by orrgottlieb — APPROVED** [review]

**Comment by orrgottlieb** [comment]
@3dyonic you have conflicts

**Pull request merged**